data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/flow-lib.xqy

```diff
@@ -456,12 +458,17 @@ declare function flow:run-plugin(
   $options as map:map)
 {
   let $module-uri := $plugin/@module
-  let $destination := $plugin/@dest
-  let $module-name := hul:get-module-name($module-uri)
-  let $ns := $PLUGIN-NS || fn:lower-case($module-name)
-  let $func := xdmp:function(fn:QName($ns, "create-" || $destination), $module-uri)
   return
-    $func($identifier, $content, $headers, $triples, $options)
+    if (fn:empty($module-uri))
```

Comment:
Rather than this fn:empty check and the one above on line 408, I'd rather you check in the caller:

line 396:

```xquery
    xdmp:invoke-function(function() {
      for $plugin in $flow/hub:plugins/hub:plugin[fn:not(hub:type = 'null')]
      let $destination := $plugin/@dest
```

Note the change:
```xquery
[fn:not(hub:type = 'null')]
```